### DISCUSSION TITLE
Why old config doesn't work with new plugins?

### DISCUSSION THREAD

**Discussion by muzhancun** [ayamir/nvimdots#701] [Q&A] [answered]
I update tree-sitter via homebrew and update all the plugins. However when I open neovim it says:
<img width="1604" alt="image" src="https://user-images.githubusercontent.com/91790440/235849793-36708096-5d3a-4b5d-b58d-eae3a64581be.png">
My config is an old version of yours, that's quite annoying

**Comment by charliie-dev** [comment, marked as answer]
Why would someone think an old config would be compatible with newer plugins?
u should debug it yourself if u choose not to keep up with the our config.
